Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -83,6 +83,33 @@ public partial class MobiledgeXIntegration
         CarrierInfoClass carrierInfoClass = new CarrierInfoClass(); // used for IsRoaming check
         MelMessaging melMessaging;
 
+        string region
+        {
+            get
+            {
+                switch (settings.region)
+                {
+                    case "EU":
+                        return EU_DME;
+                    case "KR":
+                        return KR_DME;
+                    case "JP":
+                        return JP_DME;
+                    case "US":
+                        return US_DME;
+                    case "Nearest":
+                    default:
+                        return WIFI_DME;
+                }
+            }
+        }
+
```

Comment:
This set might be sufficient for QA, but they may want to use their own staging server as well.

Reply:
I don't think we want the QA dme information to be readily available. I think the host and port overrides are a better solution

Reply:
^ True, that's what I added back. The alternate is a blank field to type in.